install_config/persistent_storage/persistent_storage_iscsi.adoc

```diff
@@ -15,7 +15,7 @@ toc::[]
 You can provision your {product-title} cluster with
 xref:../../architecture/additional_concepts/storage.adoc#architecture-additional-concepts-storage[persistent storage] using
 https://access.redhat.com/documentation/en-US/Red_Hat_Enterprise_Linux/7/html/Storage_Administration_Guide/ch-iscsi.html[iSCSI].
-Some familiarity with Kubernetes and iSCSI is assumed.
+For the purpose of this document we assume some familiarity with Kubernetes and iSCSI.
```

Comment:
"we" is first person and advised against. I'm guessing that's why this was passive int he first place. Maybe: 

"For the purpose of this document, Red Hat assumes familiarity with..."